core/src/components/progress-bar/progress-bar.ionic.scss

```
@@ -0,0 +1,31 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./progress-bar";
+
+// iOS Progress bar
```

Please make sure to update comments after copying and pasting.

```suggestion
// Ionic Progress bar
```